projects/imagenet/imagenet_experiment.py

```diff
@@ -421,6 +434,10 @@ def validate(self, loader=None):
             criterion=self.loss_function,
             batches_in_epoch=self.batches_in_epoch,
         )
+        results.update(
+            learning_rate=self.get_lr()[0],
```

Comment:
`self.lr_scheduler.get_lr()[0]`

??

Reply:
`self.get_lr()` gets the actual LR used by the optimizer

Reply:
Ah ok.